pkgs/applications/audio/bitwig-studio/bitwig-studio3.nix

```diff
@@ -1,17 +1,16 @@
-{ fetchurl, bitwig-studio1, pulseaudio, xorg }:
+{ stdenv, fetchurl, bitwig-studio2, xorg, ... }:
 
-bitwig-studio1.overrideAttrs (oldAttrs: rec {
+bitwig-studio2.overrideAttrs (oldAttrs: rec {
   name = "bitwig-studio-${version}";
-  version = "3.1.2";
+  version = "3.1.3";
 
   src = fetchurl {
-    url = "https://downloads.bitwig.com/stable/${version}/bitwig-studio-${version}.deb";
-    sha256 = "07djn52lz43ls6fa4k1ncz3m1nc5zv2j93hwyavnr66r0hlqy7l9";
+    url =
+      "https://downloads.bitwig.com/stable/${version}/bitwig-studio-${version}.deb";
```

Comment:
I'd keep this on one line.

Reply:
Can do, but this is the output of `nixfmt` I don't format my files manually.